### PR TITLE
Extra options for distrobox

### PR DESCRIPTION
**Pull request by LeSnake04**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The code compiles (`cargo build`)
- [x] The code passes rustfmt (`cargo fmt`)
- [x] The code passes clippy (`cargo clippy`)
- [x] The code passes tests (`cargo test`)
- [x] *Optional:* I have tested the code myself
    - [ ] I also tested that Topgrade skips the step where needed

If you developed a feature or a bug fix for someone else and you do not have the
means to test it, please tag this person here.

### CODE REVIEW

**Comment by LeSnake04** [comment]
@DottoDev

File: src/config.rs [hunk 1]
```diff
@@ -172,6 +172,13 @@ pub struct Windows {
     enable_winget: Option<bool>,
 }
 
+#[derive(Deserialize, Default, Debug)]
+#[serde(deny_unknown_fields)]
```
Comment (DottoDev) [inline src/config.rs:176]: missing #[allow(clippy::upper_case_acronyms)]

**Review by DottoDev — CHANGES_REQUESTED** [review]
Missing one attribute, if that's fixed it will be approved.

File: src/config.rs (hunk 1 above)
Comment (LeSnake04) [reply src/config.rs:176]: done

**Review by DottoDev — APPROVED** [review]

**Comment by LeSnake04** [comment]
I think I found the issue: I accidentally deleted the require line when refactoring for the options.

**Pull request merged**